Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "pic8259"
-version = "0.10.2"
+version = "0.10.3"
```

The version bump is still there in the latest version. We should undo it:

```suggestion
version = "0.10.2"
```